plugins/inputs/nvidia_smi/nvidia_smi.go

```diff
@@ -168,14 +177,14 @@ func setIfUsed(t string, m map[string]interface{}, k, v string) {
 			}
 		}
 	case "int":
-		if val != "" {
+		if val != "" && val != "N/A" {
 			i, err := strconv.Atoi(val)
 			if err == nil {
 				m[k] = i
 			}
 		}
 	case "str":
-		if val != "" {
+		if val != "" && val != "N/A" {
```

Comment:
CC @powersj - here `N/A` has a bit different meaning, however there were no other fields of `str` type. Again, for A10 I can see in the output:
```xml
        <retired_pages>
            <multiple_single_bit_retirement>
                <retired_count>N/A</retired_count>
                <retired_pagelist>N/A</retired_pagelist>
            </multiple_single_bit_retirement>
            <double_bit_retirement>
                <retired_count>N/A</retired_count>
                <retired_pagelist>N/A</retired_pagelist>
            </double_bit_retirement>
            <pending_blacklist>N/A</pending_blacklist>
            <pending_retirement>N/A</pending_retirement>
        </retired_pages>
```
This is because page retirement does not make sense on Ax,  this technology is [replaced by](https://support.lenovo.com/au/en/solutions/ht513682-nvidia-a100-gpu-reports-ecc-errors-lenovo-thinkagile-and-thinksystem) row remapping, but tags left in the xml.